src/display.c

```diff
@@ -1,5 +1,13 @@
 #include "CompDevSDK.h"
 
+#include <stdarg.h>
```

Comment:
Nie podoba mi się ta zmiana z noritake_display na display. Tutaj są funkcje specyficzne tylko dla jednego rodzaju wyświetlacza. Z tego co pamiętam ustalaliśmy taka architekturę:
config -> define modelu
display.h -> definicje interfejsu funkcji i include odpowiedniego headera dla modelu wyświetlacza
display_model.c -> implementacje dla konkretnych modeli zabezpieczone ifdefami

Reply:
Dobra, to wrócę do tamtego i rozdzielę na 2 osobne implementacje. Zrobiłem tak bo większość implementacji jednak jest taka sama - różnica jest w inicjalizacji.